### PR TITLE
highlight tagged metrics breaking change

### PR DESCRIPTION
**Pull request by niklibrato**
cc @chancefeick

### CODE REVIEW

**Comment by nextmat** [comment]
I don't _think_ this version actually does introduce any breaking changes for this gem, although this is true for lib-rack and lib-rails. @chancefeick ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Changelog
 
 ### Version 2.1.0
-* Add support for tagged measurements (#121)
+* Add support for tagged measurements (#121). **NOTE**: This version introduces breaking changes for legacy sources. Please contact support@librato.com to learn more.
```
Comment (chancefeick) [inline CHANGELOG.md]: This is a bit misleading, as you can still use sources. Breaking changes are only applicable to librato-rails and librato-rack, which uses either tags or sources.

Can you reword this to reflect that?
